src/components/form/datePicker.js

```diff
@@ -57,7 +60,7 @@ export const DatePicker = ({
   );
 };
 
-const DateOnlyPicker = ({ startDate, maxDate, onChange }) => {
+const DateOnlyPicker = ({ startDate, maxDate, onChange, onlyFutureTime }) => {
```

Comment:
Wouldn't be more flexible to have it as a minDate prop instead ?

Reply:
I used another field due to requirements. @aniri said that it should be a property set to true / false.
`minDate` should be a `Date` not a `Bool` by definition.

Reply:
I believe that she meant the [property in the question entry](https://github.com/code4romania/taskforce-fe-components/pull/171/files#diff-68772cae9a2fe16bd3cdefaa166253f5R15). This shouldn't be to different from what is already implemented for `allowFuture` in the previous link.

Reply:
Sorry for my mistake,. I will rework the code tonight.

Reply:
yes, @surdu is right, sorry for the confusion. as I mentioned in #173 , the implementation should be similar to the one for `allowFuture`